src/protocols/secure_channel/Constants.h

```diff
@@ -42,6 +43,8 @@ namespace chip {
 namespace Protocols {
 namespace SecureChannel {
 
+constexpr const char * kName = "SecureChannel";
```

Comment:
All the `kName` globally at the top of the namespaces should probably by `kProtocolName`